app/Http/Controllers/SitemapController.php

```diff
@@ -22,17 +22,17 @@ public function index(){
         return Sitemap::index();
     }
 
-    Public function users(){
+    Public function users() {
```

Comment:
__[code_sniffer]__
* PHP keywords must be lowercase; expected "public" but found "Public"
* PUBLIC keyword must be lowercase; expected "public" but found "Public"
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->